### ISSUE TITLE
Support node 22

### ISSUE THREAD

**Issue by chenrui333** [lando/cli#272]
👋 While reviewing [lando-cli 3.21.2 (new formula) PR](https://github.com/Homebrew/homebrew-core/pull/175439), I found that lando-cli is still using node 18, any plan to update to node 22? Thanks!

**Comment by AaronFeledy** [comment]
@chenrui333 This repo is actually the source for the CLI component of the Lando product that ships as a compiled binary. Users shouldn't be installing it as something to run via node.  I left a note on the PR you linked.

**Comment by chenrui333** [comment]
yeah, all the homebrew formulae are built from source. but you dont think we built the formulae/binary artifacts in the right way, I dont quite get it. Thanks!
